LSL/liblsl/CMakeLists.txt

```diff
@@ -1,36 +1,165 @@
-cmake_minimum_required (VERSION 2.6)
-project (liblsl)
-
-# Version number
-set (liblsl_VERSION_MAJOR 1)
-set (liblsl_VERSION_MINOR 4)
-set (liblsl_VERSION_PATCH 0)
+cmake_minimum_required (VERSION 3.5)
+project (LSL
+	VERSION 1.4.0
+	LANGUAGES C CXX)
 
 cmake_policy(SET CMP0042 NEW) # ENABLE CMP0042: MACOSX_RPATH is enabled by default.
 if(NOT CMAKE_DEBUG_POSTFIX)
-  set(CMAKE_DEBUG_POSTFIX -debug)
+	set(CMAKE_DEBUG_POSTFIX -debug)
 endif()
-SET( CMAKE_ARCHIVE_OUTPUT_DIRECTORY         ${CMAKE_CURRENT_LIST_DIR}/bin )
```

Comment:
If the generated files should be placed in the source directory, this should be specified only for the target in `install()`